### ISSUE TITLE
Please note project abandonment

### ISSUE THREAD

**Issue by Traumflug** [mfkp/tinderplusplus#44]
Unless I'm severly mistaken, this project is abandoned in favour of [tinder-desktop](https://github.com/tinderjs/tinder-desktop)

It's entirely fine to abandon projects and to continue elsewhere, but please, please note this in the toplevel README.md so people can see it. Without such a note people might try this one ... and fail. Just wasted the better part of two days trying to get this one running.

**Comment by mfkp** [comment]
Sorry about the trouble - been traveling and swamped with work lately. Can you let me know what problem you had getting it running? (I just fixed a couple things pushing out a new version since some of the dependencies changed with the newer node versions).

Thanks!

**Comment by Traumflug** [comment]
Oh, it's expected to work :-) Nice surprise! I'll try again and take more detailed notes ... which likely takes some time on my side.

**Comment by macandchief** [comment]
Yes, same here... I got so far that the main window opened with the facebook button. After clicking on it an other window opened with no content. Tried on Ubuntu. If I remember correctly, also node-webkit-builder is required (its name changed meanwhile), but it's not mentioned in the readme file.

**Comment by rebroad** [comment]
yes, doing a diff of the logs of both projects, tinder-desktop is clearly a fork of tinderplusplus, but with significant further development, whereas tinderplusplus has only 3 relatively minor commits following the fork.
